File: tarea 2/python/ejercicio_7.py
# 7. Buscador de Divisores y Primalidad

N = int(input("Ingrese un numero: "))
contador = 0

print("Divisores:")

for i in range(1, N + 1):
    if N % i == 0:
        print(i)
        contador += 1

if contador == 2:
    print("El numero es primo")
else:
    print("El numero no es primo")
    input("presione par salir")
    

File: tarea 2/c++/ejercicio 7.cpp
// 7. Buscador de Divisores y Primalidad
#include <iostream>
using namespace std;

int main() {
    int N, contador = 0;

    cout << "Ingrese un numero: ";
    cin >> N;

    cout << "Divisores: ";

    for (int i = 1; i <= N; i++) {
        if (N % i == 0) {
            cout << i << " ";
            contador++;
        }
    }

    cout << endl;

    if (contador == 2)
        cout << "El numero es primo";
    else
        cout << "El numero no es primo";

    return 0;
}

